packages/discord.js/src/managers/GuildSoundboardSoundManager.js

```diff
@@ -0,0 +1,192 @@
+ * @extends {CachedManager}
+ */
+class GuildSoundboardSoundManager extends CachedManager {
+  constructor(guild, iterable) {
+    super(guild.client, SoundboardSound, iterable);
+
+    /**
+     * The guild this manager belongs to
+     * @type {Guild}
+     */
+    this.guild = guild;
+  }
+
+  /**
+   * The cache of Soundboard Sounds
+   * @type {Collection<Snowflake, SoundboardSound>}
+   * @name GuildSoundboardSoundManager#cache
+   */
+
+  _add(data, cache) {
+    return super._add(data, cache, { extras: [this.guild], id: data.sound_id });
+  }
+
+  /**
+   * Data that resolves to give a SoundboardSound object. This can be:
+   * * A SoundboardSound object
+   * * A Snowflake
+   * @typedef {SoundboardSound|Snowflake} SoundboardSoundResolvable
+   */
+
+  /**
+   * Resolves a SoundboardSoundResolvable to a SoundboardSound object.
+   * @method resolve
+   * @memberof GuildSoundboardSoundManager
+   * @instance
+   * @param {SoundboardSoundResolvable} soundboardSound The SoundboardSound resolvable to identify
+   * @returns {?SoundboardSound}
+   */
+
+  /**
+   * Resolves a {@link SoundboardSoundResolvable} to a {@link SoundboardSound} id.
+   * @param {SoundboardSoundResolvable} soundboardSound The soundboard sound resolvable to resolve
+   * @returns {?Snowflake}
+   */
+  resolveId(soundboardSound) {
+    if (soundboardSound instanceof this.holds) return soundboardSound.soundId;
+    if (typeof soundboardSound === 'string') return soundboardSound;
+    return null;
+  }
+
+  /**
+   * Options used to create a soundboard sound in a guild.
+   * @typedef {Object} GuildSoundboardSoundCreateOptions
+   * @property {BufferResolvable|Stream} file The file for the soundboard sound
+   * @property {string} name The name for the soundboard sound
+   * @property {string} [contentType] The content type for the soundboard sound file
+   * @property {number} [volume] The volume for the soundboard sound, from 0 to 1. Defaults to 1
+   * @property {Snowflake} [emojiId] The emoji id for the soundboard sound
+   * @property {string} [emojiName] The emoji name for the soundboard sound
+   * @property {string} [reason] The reason for creating the soundboard sound
+   */
+
+  /**
+   * Creates a new guild soundboard sound.
+   * @param {GuildSoundboardSoundCreateOptions} options Options for creating a guild soundboard sound
+   * @returns {Promise<SoundboardSound>} The created soundboard sound
+   * @example
+   * // Create a new soundboard sound from a file on your computer
+   * guild.soundboardSounds.create({ file: './sound.mp3', name: 'sound' })
+   *   .then(sound => console.log(`Created new soundboard sound with name ${sound.name}!`))
+   *   .catch(console.error);
+   */
+  async create({ contentType, emojiId, emojiName, file, name, reason, volume }) {
+    const resolvedFile = await resolveFile(file);
+
+    const resolvedContentType = contentType ?? resolvedFile.contentType ?? fileTypeMime()(resolvedFile.data)[0];
```

Comment:
Should probably assert that you get a content type (maybe one that is also valid?)